src/components/ToolTip.svelte

```diff
@@ -0,0 +1,43 @@
+<script>
+    export let title = '';
+    let isHovered = false;
+    let x;
+    let y;
+
+    const mouseOver = (event) => {
+        isHovered = true;
+        x = event.pageX + 5;
+        y = event.pageY + 5;
+    }
+
+    const mouseMove = (event) => {
+        x = event.pageX + 5;
+        y = event.pageY + 5;
+    }
+
+    const mouseLeave =() => {
+        isHovered = false;
+    }
+</script>
+
+<div
+        on:mouseover={mouseOver}
+        on:mouseleave={mouseLeave}
+        on:mousemove={mouseMove}>
+    <slot/>
+</div>
+
+{#if isHovered}
+    <div style="top: {y}px; left: {x}px;" class="tooltip">{@html title}</div>
+{/if}
+
+<style>
```

Comment:
Style tags usually come before the content they're affecting. Feel free to resolve this comment without making changes

Reply:
It's common for style tags in svelte to appear at the bottom of the file. These style tags are scoped to the component

Reply:
Ah, very cool. Carry on